src/transformers/pipelines/audio_utils.py

```diff
@@ -246,3 +244,13 @@ def _get_microphone_name():
         print("ffmpeg was not found. Please install it or make sure it is in your system PATH.")
 
     return "default"
+
+
+def _ffmpeg_format_for_platform(platform: str):
+    platform_to_format_map = {"Linux": "alsa", "Darwin": "avfoundation", "Windows": "dshow"}
+    return platform_to_format_map[platform]
+
+
+def _ffmpeg_input_device_for_platform(platform: str):
+    platform_to_input_device_map = {"Linux": "default", "Darwin": ":default", "Windows": _get_microphone_name()}
+    return platform_to_input_device_map[platform]
```

Comment:
The previous if/else logic was better: we only want to run `_get_microphone_name()` when we're running on windows

Reply:
Oops! Bit overzealous there, will revert.